src/positions/slice.ts

```diff
@@ -22,6 +22,7 @@ export type TriggeredShortcuts = Record<
 interface State {
   positions: Position[]
   positionsFetchedAt?: number
+  earnPositionIds: string[]
```

Comment:
We can also store this in the earn state, but I guess its easier to put this here since it all comes from a single saga

Reply:
Yes I was hesitating.